packages/dendron-next-server/pages/_app.tsx

```diff
@@ -79,6 +79,8 @@ function AppVSCode({ Component, pageProps }: any) {
     logger.info("AppVSCode.init");
   }, []);
 
+  // query only has ws and port properties. To intitalize engine with useEngine, we require url as well.
+  query.url = `http://localhost:${query.port}`;
```

Comment:
```suggestion
  query.url = workspaceOpts?.url;
```

Since we are already calling `getWorkspaceParamsFromQueryString` in the initial hook this works.